vllm/utils.py

```diff
@@ -543,11 +544,37 @@ def get_open_zmq_ipc_path() -> str:
     return f"ipc://{base_rpc_path}/{uuid4()}"
 
 
-def get_open_port() -> int:
+def get_open_port(force: bool = False) -> int:
     port = envs.VLLM_PORT
+
+    if force:
+        # This flag will only be True in disaggregated prefill scenario
+        # and it has to be set so that vLLM can connect prefill vLLM instance
+        # and decode vLLM instance.
+        assert port is not None, "Please set environment variable VLLM_PORT in"
+        " order to use disaggregated prefill and distributed KV cache transfer"
+
+        # For prefill vLLM instance (KV producer) this port must be empty.
+        # For decode vLLM instance this port can be non-empty.
```

Comment:
The term `empty` is a bit confused. Do you mean "available" or "not in-use"?

Reply:
Yes it is a bit confusing. I changed it to available.